corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -69,12 +74,18 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
                             "></select>
                         </td>
                         <td>
-                            <textarea data-bind="
+                            <textarea class="form-control" data-bind="
                               value: $data.description,
                               rows: 1,
                               placeholder: '{% trans "Click here to add a description" %}',
                             "></textarea>
                         </td>
+                        <!-- ko if: !deprecated() -->
+                        <td><button data-bind="click: deprecateProperty" class="btn btn-danger">{% trans "Deprecate Property" %}</button></td>
+                        <!-- /ko -->
+                        <!-- ko if: deprecated() -->
+                        <td><button data-bind="click: restoreProperty" class="btn btn-success">{% trans "Restore Property" %}</button></td>
```

Comment:
I think these might be better with:
* deprecate: `btn btn-warning btn-sm`
* restore: `btn btn-info btn-sm`